### PR TITLE
pass slug to kebechet workflow

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

https://github.com/thoth-station/thoth-application/pull/987

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

One thing to note is that as of now we assume our web-hooks are coming from GitHub

### CODE REVIEW

File: thoth/common/openshift.py [hunk 1]
```diff
@@ -1516,9 +1516,13 @@ def schedule_kebechet_workflow(
     ) -> Optional[str]:
         """Schedule Kebechet Workflow for a Webhook from GitHub App.."""
         workflow_id = job_id or self.generate_id("kebechet-job")
+        url = webhook_payload["repository"]["html_url"]
```
Comment (goern) [inline thoth/common/openshift.py:1519]: are you looking for pull_request.base.repo.full_name?

Comment (KPostOffice) [reply thoth/common/openshift.py:1519]: Ummm, not sure what `pull_requrest.base.repo.full_name` is. I'm looking for `https://github.com/<foo>/<bar>` and then I am trying to extract `<foo>/<bar>`

Comment (harshad16) [reply thoth/common/openshift.py]: the webhook_payload["repository"]["full_name"] might be just enough as @goern suggested.

Comment (goern) [reply thoth/common/openshift.py]: ja, right :)

**Review by harshad16 — COMMENTED** [review]
/lgtm

**Review by pacospace — APPROVED** [review]
lgtm!

**Pull request merged**